installer/charts/rhtap-dh/templates/extra-env.yaml

```diff
@@ -63,3 +63,9 @@ data:
     QUAY__API_TOKEN: {{ $quaySecretData.token }}
 {{- end }}
     QUAY__URL: {{ $quaySecretData.url }}
+{{- $nexusSecretObj := (lookup "v1" "Secret" .Release.Namespace "rhtap-nexus-integration") }}
+{{- $nexusSecretData := ($nexusSecretObj.data | default dict) }}
+{{- if $nexusSecretData }}
+    NEXUS__TOKEN: {{ $nexusSecretData.token }}
```

Comment:
Are there public repositories on Nexus? If so the token might be optional. You would need to update `nexus.go` too.

Reply:
I am also looking for this but so far couldn't find a way out.